Find a functionally equivalent code:android.widget.CalendarView.setFirstDayOfWeek:COMMENT
Method Modifier: public      
Comment:/**
 * Sets the first day of week.
 *
 * @param firstDayOfWeek The first day of the week conforming to the
 * {@link CalendarView} APIs.
 * @see Calendar#MONDAY
 * @see Calendar#TUESDAY
 * @see Calendar#WEDNESDAY
 * @see Calendar#THURSDAY
 * @see Calendar#FRIDAY
 * @see Calendar#SATURDAY
 * @see Calendar#SUNDAY
 *
 * @attr ref android.R.styleable#CalendarView_firstDayOfWeek
 */

@@ -1,8 +1,3 @@
 {
-    if (mFirstDayOfWeek == firstDayOfWeek) {
-        return;
-    }
-    mFirstDayOfWeek = firstDayOfWeek;
-    mAdapter.init();
-    setUpHeader();
+    mDelegate.setFirstDayOfWeek(firstDayOfWeek);
 }

